services/src/test/java/io/druid/cli/validate/DruidJsonValidatorTest.java

```diff
@@ -68,6 +68,41 @@ public void testExceptionCaseNoFile()
     druidJsonValidator.run();
   }
 
+  @Test(expected = RuntimeException.class)
+  public void testParseValidatorInvalid()
+  {
+    String type = "parse";
+    Cli<?> parser = Cli.builder("validator")
+        .withCommand(DruidJsonValidator.class)
+        .build();
+    Object command = parser.parse(
+        "validator",
+        "-f", "simple_test_data_record_parser_invalid.json",
+        "-t", type
+    );
+    Assert.assertNotNull(command);
+    DruidJsonValidator druidJsonValidator = (DruidJsonValidator) command;
+    druidJsonValidator.run();
+  }
+
+  @Test
+  public void testParseValidator()
+  {
+    String type = "parse";
+    Cli<?> parser = Cli.builder("validator")
+        .withCommand(DruidJsonValidator.class)
+        .build();
+    Object command = parser.parse(
+        "validator",
+        "-f", "simple_test_data_record_parser.json",
+        "-r", "simple_test_data.tsv",
```

Comment:
Can you cut the data size down small enough so that you can have an explicit string in here for what its supposed to return?